conf/rest/9.12.0/security_audit_dest.yaml

```diff
@@ -4,7 +4,9 @@ query:            api/security/audit/destinations
 object:           security_audit_destination
 
 counters:
-  - ^protocol     => protocol
+  - ^^protocol                   => protocol
+  - ^^address                    => destination
```

Comment:
^^ == instance key but not listed in `instance_keys` section below?

Reply:
`only_cluster_instance` flag rest changes will be done later, reverted those changes.